package/src/contexts/messageInputContext/MessageInputContext.tsx

```diff
@@ -70,7 +70,7 @@ export type ImageUpload = {
     name?: string;
   };
   id: string;
-  state: string;
+  state: typeof FileState[keyof typeof FileState];
```

Comment:
```suggestion
  // at the top of the file
  type FileStateValues = typeof FileState[keyof typeof FileState];
  // then
  state: FileStateValues; // shared at multiple places in the file, so
```

This is called "Don't repeat yourself" (DRY) principle. If interested, you can read more about it.

Reply:
Good point. `FileStateValue` is a better name.

Reply:
✅